### ISSUE TITLE
Docker : "build-only-webapp" option fail  (root cause: jdeb 1.6)

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Contrast-Finder#166]
## Observed behaviour
```
[INFO] Attaching created debian package /home/fabrice/Bureau/Asqatasun/git.contrast-finder.build/webapp/ta
rget/contrast-finder-webapp_0.9.2-SNAPSHOT_tomcat8.deb
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ webapp ---
[WARNING] Error injecting: org.apache.maven.artifact.installer.DefaultArtifactInstaller
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting: private org.eclipse.aether.spi.log.Logger org.apache.maven.repository.internal.Default
VersionRangeResolver.logger
  while locating org.apache.maven.repository.internal.DefaultVersionRangeResolver
  while locating java.lang.Object annotated with *
  at org.eclipse.sisu.wire.LocatorWiring
  while locating org.eclipse.aether.impl.VersionRangeResolver
    for parameter 2 at org.eclipse.aether.internal.impl.DefaultDependencyCollector.<init>(DefaultDependencyCollector.java:92)
```

## Steps to reproduce the behaviour
```
mvn clean install
cd webapp/
mvn clean install
```
or
```
./docker/build_and_run-with-docker.sh                     --source-dir $(pwd) --docker-dir docker/SNAPSHOT-local 
./docker/build_and_run-with-docker.sh --build-only-webapp --source-dir $(pwd) --docker-dir docker/SNAPSHOT-local 
```

**Comment by dzc34** [comment]
bug present after 8.3 release
caused by  :  https://github.com/Asqatasun/Contrast-Finder/commit/83f724e9730d1170389727b4c30bf327e4929cec _"Fixed #118 - Upgraded outdated dependency: jdeb (1.6 instead of 1.5)"_

**Comment by dzc34** [comment]
c7a92fd 2 fixes the bug, 
but causes an error for the `mvn site` command

```bash
mvn site

(...)
[INFO] Contrast-Finder .................................... SUCCESS [ 13.364 s]
[INFO] contrast-finder-documentation ...................... SUCCESS [  2.663 s]
[INFO] contrast-finder-engine ............................. SUCCESS [  3.203 s]
[INFO] utils .............................................. SUCCESS [  6.402 s]
[INFO] api ................................................ SUCCESS [  6.504 s]
[INFO] impl ............................................... SUCCESS [  6.339 s]
[INFO] hsv ................................................ SUCCESS [  7.606 s]
[INFO] contrast-finder-webapp ............................. SUCCESS [ 18.264 s]
[INFO] contrast-finder-dist_debian-packages ............... FAILURE [  0.129 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:04 min
[INFO] Finished at: 2018-07-01T21:42:36+02:00
[INFO] Final Memory: 84M/1527M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) 
on project contrast-finder-dist_debian-packages: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed: 
A required class was missing while executing org.apache.maven.plugins:maven-site-plugin:3.3:site: org/apache/maven/doxia/siterenderer/DocumentContent
```
see: [Maven skip site/reporting for module / stackoverflow](https://stackoverflow.com/a/10841119)

add to `dist./debian_package/pom.xml`:
```xml
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-site-plugin</artifactId>
                <configuration>
                    <skip>true</skip>
                </configuration>
            </plugin>
```
